gradle/libs.versions.toml

```diff
@@ -67,7 +67,7 @@ hibernateCore = { group = "org.hibernate", name = "hibernate-core", version.ref
 hibernateJCache = { group = "org.hibernate", name = "hibernate-jcache", version.ref = "hibernateVersion" }
 # Hibernate up until 6.6 is only compatible with Jakarta Persistence 3.1
 jakartaPersistence = { group = "jakarta.persistence", name = "jakarta.persistence-api", version = "3.1.0" }
-hibernateCommonsAnnotations = { group = "org.hibernate.common", name = "hibernate-commons-annotations", version = "7.0.3.Final" }
+hibernateCommonsAnnotations = { group = "org.hibernate.common", name = "hibernate-commons-annotations", version = "7.0.3.Final-redhat-00001" }
```

Comment:
I'm not sold on the benefits of the RedHat packages. I have never found what the differences are that they make to the originals. Today I tried again, I cannot find their code, I cannot find changelogs, I just waste time.

If up to me: I'd say let's remove the redhat maven repo and be done with this. But that maybe my shortsightedness because I simply do not know the benefits.

We have only 2 packages from that repo now.

Also the redhat repo is contagious: all RePlay projects need to add it as well, or RePlay's transient dependencies cannot be found.